client/src/pages/LoginPage/LoginPage.js

```diff
@@ -57,11 +73,7 @@ export default function LoginPage() {
           </Link>
         </div>
         <Link to="/register">
-          <button
-            className="loginPageButton"
-            type="button"
-            onClick={handleSubmit}
-          >
+          <button className="loginPageButton" type="button">
```

Comment:
why is the `onClick` gone?

Reply:
the previous function, handleSubmit, that was on the button, only cleared the user input fields.
We decided that clearing the input fields when the register button is pressed is unnecessary because it brings
us to a new page.

Reply:
don't we need to call `checkLogin` though?

Reply:
+1

Reply:
checkLogin will be called in the handleSubmit, which is used when 'Login" button is clicked, not when 'Create Account' button is clicked. checkLogin only checks if the inputted email and password are a valid combination found in the database, so we thought it does not make sense to check this if the 'Create Account' button was clicked